superset-frontend/src/views/components/MenuRight.tsx

```diff
@@ -163,13 +186,17 @@ const RightMenu = ({
                     className="data-menu"
                     title={menuIconAndLabel(menu)}
                   >
-                    {menu.childs.map(item =>
+                    {menu.childs.map((item, idx) =>
                       typeof item !== 'string' &&
                       item.name &&
                       configMap[item.name] === true ? (
-                        <Menu.Item key={item.name}>
-                          <a href={item.url}> {item.label} </a>
-                        </Menu.Item>
+                        <>
+                          {idx === 2 && <Menu.Divider />}
```

Comment:
if we go above idx > 1, this would render after every menu item after the first 2 items. We just want it to render it once right after the db connection buttons. Thats the one of the main reason i split out the db connection buttons from the config.

Reply:
Ok, I see the complication. Again I would suggest separating the concerns between the data at the top and the rendering at the bottom. You could add the logic to the data with something like:

```
      {
          label: t('Connect Google Sheet'),
          name: GlobalMenuDataOptions.GOOGLE_SHEETS,
          perm: HAS_GSHEETS_INSTALLED,
        },
       '---------',
        {
          label: t('Upload a CSV'),
          name: 'Upload a CSV',
          url: '/csvtodatabaseview/form',
          perm: CSV_EXTENSIONS,
        },
```
and then you can render the divider if typeof item === 'string'
or make it an object with a type of "divider"...

Reply:
This is something we can discuss later.